cmd/swapd/main.go

```diff
@@ -675,11 +676,9 @@ func getProtocolInstances(c *cli.Context, cfg common.Config,
 	walletPassword := c.String(flagMoneroWalletPassword)
 
 	xmrtakerCfg := &xmrtaker.Config{
-		Backend:              b,
-		DataDir:              cfg.DataDir,
-		MoneroWalletFile:     walletFilePath,
-		MoneroWalletPassword: walletPassword,
```

Comment:
why were these removed?

Reply:
Yes, they were no longer needed here, as we now cache the primary address and information to re-open the primary wallet in the `walletClient` struct.